src/models/activityReportObjectiveCitation.js

```diff
@@ -51,6 +51,29 @@ export default (sequelize, DataTypes) => {
       allowNull: false,
       defaultValue: DataTypes.NOW,
     },
+    findingIds: {
```

Comment:
So we track these here instead of the monitoring ref?

Reply:
its a virtual column for convenience